src/main/scala/Afe.scala

```diff
@@ -0,0 +1,94 @@
+package edu.berkeley.cs.ucie.digital
+
+import chisel3._
+import chisel3.util._
+
+class StandardPackageIo(lanes: Int = 16) extends Bundle {
```

Comment:
It would probably be better to declare the mainband and sideband wires as separate bundles, and then have a TX and RX instance inside `StandardPackageIo` - this way, we can use Chisel bulk-connection operators to easily wire up a pair (e.g. `a.mainbandTx <> b.mainbandRx`)

Reply:
Is there ever a case where you'd want to connect the MB and SB separately? I think we'd always do `a.tx <> b.rx` rather than `a.mainbandTx <> b.mainbandRx` and `a.sidebandTx <> b.sidebandRx`.

Reply:
We can combine both of these by having `a.tx` contain `mainband` and `sideband` bundles :) Then you have the convenience of connecting both mainband and sideband in the normal case, but can connect them independently if something requires it.